commands/activity.js

```diff
@@ -0,0 +1,41 @@
+import fs from 'node:fs/promises'
+import { Tail } from 'tail'
+import { paths } from '../lib/paths.js'
+import { parseLog, formatLog } from '../lib/log.js'
+
+const formatLogLine = line => {
+  const { date, text } = parseLog(line)
+  const { message } = JSON.parse(text)
+  return formatLog(message, date)
```

Comment:
I think we should add an indication of whether the activity is INFO or ERROR.

For example (notice the additional space after `INFO` to keep things aligned):

```
[3/14/2023, 10:23:14 AM] INFO  Saturn Node is online and connected to 1 peers
[3/14/2023, 10:23:14 AM] ERROR Saturn Node cannot connect to the Orchestrator
```